source/learn/intrinsics/_pages/ACHAR.md

```diff
@@ -161,13 +161,13 @@ FORTRAN 77 and later, with KIND argument Fortran 2003 and later
 
 ### **See Also**
 
-[**char**(3)](#char),
-[**iachar**(3)](#iachar),
-[**ichar**(3)](#ichar)
+[**char**(3)](CHAR),
+[**iachar**(3)](IACHAR),
+[**ichar**(3)](ICHAR)
```

Comment:
This change is going to revert the cross-linking introduced with #152

Reply:
I deleted the old one and made a new pull request. Not sure how the old text got there. Going to delete this and
start over with a new one.  Will just do a single one for the footer line change so only a few files are in the other change